nucypher/blockchain/eth/signers/base.py

```diff
@@ -90,5 +95,5 @@ def sign_transaction(self, transaction_dict: dict) -> HexBytes:
         return NotImplemented
 
     @abstractmethod
-    def sign_message(self, account: str, message: bytes, **kwargs) -> HexBytes:
+    def sign_message(self, account: str, message: bytes, **kwargs) -> SignedMessage:
```

Comment:
I don't agree much with this change. As I understand it, `SignedMessage.signer == account` will always hold, making it totally redundant. Returning `HexBytes` seems to me like the cleanest return type.

Reply:
:heavy_check_mark: